ga4gh/exceptions.py

```
@@ -0,0 +1,181 @@
+"""
+Exceptions for the GA4GH server. Each exception that can occur in the server
+is given a unique error code that is derived from its name.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import zlib
+
+import ga4gh.protocol as protocol
+
+
+def getServerError(exception):
+    """
+    Converts the specified exception that is not a subclass of
+    BaseServerException into a ServerError so that it can be correctly
+    serialised and communicated back to the client.
+    """
+    assert not isinstance(exception, BaseServerException)
```

can we raise a specific error here?  I'm kind of allergic to the assert keyword